test/common/http/conn_manager_impl_test_2.cc

```diff
@@ -3132,60 +3131,10 @@ traffic_direction: INBOUND
   conn_manager_->onData(fake_input, false);
 }
 
-TEST_F(HttpConnectionManagerImplTest, InboundOnlyDrainNoConnectionCloseForOutbound) {
```

Comment:
what's the reason to delete this test?

Reply:
I felt like we already did this in the admin test; I can add it back if you like

Reply:
Added